pyprocessing/color.py

```diff
@@ -123,20 +138,46 @@ def lerp_color(color_from, color_to, amount):
 
 # Setting colors
 
-def stroke(color=0):
+def stroke(*colors):
     pp = PyProcessing()
-    color = Color(color)
+    color = Color(*colors)
     pp.namespace['stroke'] = color
 
 
-def fill(color=255):
+def no_stroke():
     pp = PyProcessing()
-    color = Color(color)
+    pp.namespace['stroke'] = None
+
+
+def fill(*colors):
+    pp = PyProcessing()
+    color = Color(*colors)
     pp.namespace['fill'] = color
 
 
+def no_fill():
+    pp = PyProcessing()
+    pp.namespace['fill'] = None
+
+
 def background(color):
     pp = PyProcessing()
     color = Color(color)
-
     pp.windows.set_background(color)
+
+
+def color_mode(mode, *args):
+    if mode != 1 and mode != 3:
```

Comment:
use an Enum for that, or use the Color.RBG / Color.HSV constants to make the code self-documenting